src/Logging/Logging.Batching/BatchLogService.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace CodeMonkeys.Logging.Batching
+{
+    public abstract class BatchLogService<TOptions> : ScopedLogService<TOptions>
+        where TOptions : BatchLogOptions, new()
+    {
+        private BlockingCollection<LogMessage> _queue;
+
+        public override bool IsEnabled
+        {
+            get => base.IsEnabled;
+            protected set
+            {
+                if (value)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5626772)